### PR TITLE
Calculating anomaly_score and sending notifications for user models

### PR DESCRIPTION
**Pull request by Knispel2**

### CODE REVIEW

File: fink_broker/science.py [hunk 1]
```diff
@@ -410,10 +411,12 @@ def apply_science_modules(df: DataFrame, tns_raw_output: str = "") -> DataFrame:
     ]
 
     df = df.withColumn("lc_features", extract_features_ad(*ad_args))
-
     # Apply level one processor: anomaly_score
     _LOG.info("New processor: Anomaly score")
-    df = df.withColumn("anomaly_score", anomaly_score("lc_features"))
+    ANOMALY_MODELS = [''] + ANOMALY_MODELS # '' - model for a public channel
```
Comment (JulienPeloton) [inline fink_broker/science.py:416]: In Python, global scope variable cannot not be used for assignment inside a local scope.

```python
# global definition
a = 1

def f():
  a = a +1
  return a
print(f())
```

will lead to 

```
UnboundLocalError: local variable 'a' referenced before assignment
```

So you need to define a new variable name for the expanded list of models.

**Review by JulienPeloton — CHANGES_REQUESTED** [review]
Thanks @Knispel2  -- could you fix PEP8 complains?

File: fink_broker/science.py (hunk 1 above)
Comment (Knispel2) [reply fink_broker/science.py]: Sorry, silly mistake, I overlooked it

**Comment by Knispel2** [comment]
Sorry, using ruff is unusual for me

**Comment by JulienPeloton** [comment]
No worries -- just installing `ruff==0.8.4` and running:

```bash
ruff check --preview fink_broker/*.py
ruff format --check --preview fink_broker/*.py
ruff check --preview bin/*.py
ruff format --preview bin/*.py
```

Should lint the code correctly (see https://github.com/astrolabsoftware/fink-broker/blob/master/.github/workflows/linter.yml)

**Comment by Knispel2** [comment]
> ```shell
> ruff format --preview bin/*.py
> ```
I get this:

image

**Comment by JulienPeloton** [comment]
All good now :-)

**Comment by Knispel2** [comment]
Thanks, `ruff format --preview` helped

**Pull request merged**